web/js/index.js

```diff
@@ -229,12 +244,12 @@ $elemSelector.addEventListener('change', async (evt) => {
             });
         } catch (err) {
             console.error('Error requesting GLTF data translation', err);
-                displayError(`Error requesting GLTF data translation: ${err}`);
+            displayError(`Error requesting GLTF data translation: ${err}`);
```

Comment:
As far as I've seen, every time we call displayError, we also call console.error. Maybe just put console.error call in displayError in to avoid duplication?

Reply:
Updated.